app/controllers/projects_controller.rb

```diff
@@ -2,16 +2,20 @@ class ProjectsController < ApplicationController
   def user_project_metric
     return if metric_params.blank?
 
-    @metrics = Queries::BaseQueryMetric.determinate_metric_period(metric_params[:period]).call(
-      project_id: project_id,
-      metric_name: metric_params[:name]
+    period_metric_query = Queries::PeriodMetricRetriever.call(metric_params[:period])
+
+    @review_turnaround = period_metric_query.call(
+      project_id: project_id, metric_name: 'review_turnaround'
+    )
+    @merge_time = period_metric_query.call(
+      project_id: project_id, metric_name: 'merge_time'
     )
   end
 
   private
 
   def project_id
-    @project_id ||= Project.find_by(name: metric_params[:project_name]).id
+    @project_id ||= Project.find_by(name: params[:project_name]).id
   end
 
   def metric_params
```

Comment:
Can we move the hole controller to UsersProjectController? And keep the URL like this `/users_project/metrics` ?
Also, I am working on Project metrics, so I will need to create another method called `project_metrics` which I found a little redundant.

Reply:
of course, few minutes and i will upload the changes